docs/1-getting-started/4-making-a-blog/README.md

```diff
@@ -0,0 +1,208 @@
+
+![Blog post edition interface](new-blog-post.png)
+
+## Listing Articles
+
+Let's get back to the **Pages** tab of the sidebar. Create a new page named "Blog" with a **Single column** layout. Create a Hero section on top of the page to introduce the list of articles.
+
+In the main area, add a new **Jahia - Queries > Content items using JCR Query** component. The JCR is the tree-like database where all your content is stored, pages and content folders alike. We will use a JCR query to list all the blog posts. In the query field, enter:
+
+```sql
+SELECT *
+FROM [hydrogen:blogPost] AS post
+WHERE ISDESCENDANTNODE(post, '/sites/hydrogen/contents/blog')
+```
+
+JCR-SQL2 is a SQL-like language to query the JCR. This query selects all nodes of type `hydrogen:blogPost` that are descendants of the `/sites/hydrogen/contents/blog` node. In this query, `hydrogen` is the site key, you might have set it to something else when creating your site.
+
+![JCR query edition interface](jcr-query.png)
+
+You can set _Items to show_ to a positive number to limit the number of articles displayed, or -1 to show all of them.
+
+You can also add a rich text above and below the list of articles to introduce and conclude the blog page, and update the footer to reference your Blog page. When done, you should have a page that looks like this:
+
+![Preview of the Blog page](blog-page.png)
+
+[Cover image by RetroSupply on Unsplash](https://unsplash.com/photos/vintage-teal-typewriter-beside-book-jLwVAUtLOAQ), article images by [Belinda Fewings on Unsplash](https://unsplash.com/photos/3d-painting-of-welcome-6wAGwpsXHE0) and [Olga Tutunaru on Unsplash](https://unsplash.com/photos/white-book-page-on-white-textile-plbb7pkEjkQ).
+
+If you click on a blog post, you'll end up on a 404 error page. We'll take care of that in the next section. But for now...
+
+## Publication Date
+
+Our JCR query is pretty simple: it lists all blog posts. However, our posts are not ordered and we don't have a way to publish them at a specific date. Let's add a `publicationDate` field to our `BlogPost` type:
+
+```cnd
+ - publicationDate (date)
+```
+
+It's not marked as `mandatory` because we can't add a mandatory field to an existing type, it would be a breaking change. We will consider it a feature: if the date is not set, the post is a draft.
+
+You also need to update the `types.ts` file:
+
+```ts
+import type { JCRNodeWrapper } from "org.jahia.services.content";
+
+export type Props = {
+  "jcr:title": string;
+  "subtitle": string;
+  "authors"?: string[];
+  "cover": JCRNodeWrapper;
+  "body": string;
+  "publicationDate": string;
+};
+```
+
+`publicationDate` is a string because it's a date in the ISO 8601 format. As the native `Date` object does not properly handle time zones, it's easier to handle dates as strings if your application requires time zone support.
+
+We can also update our JCR query to order the posts by publication date, and only show published posts:
```

Comment:
Perhaph's we could reming the user they have to set a publication date on their blog posts in order to show up on the Blog page?